### ISSUE TITLE
Crash on STDIO doors when logged in via WebSocket

### ISSUE THREAD

**Issue by apamment** [NuSkooler/enigma-bbs#118]
For :bug: bug reports, please fill out the information below plus any additional relevant information. If this is a feature request, feel free to clear the form.

**Short problem description**

**Environment**
- [X] I am using Node.js v6.x or higher
- [X] `npm install` reports success
- Actual Node.js version (`node --version`): v6.11.2
- Operating system (`uname -a` on *nix systems): Ubuntu 16.04 amd64
- Revision (`git rev-parse --short HEAD`):9569691

**Expected behavior**
Door should work
**Actual behavior**
Enigma Crashes with this on the console:
```
TypeError: self.client.term.output.pipe is not a function
    at prepareSocketIoServer (/home/andrew/enigma-bbs/core/door.js:125:28)
    at Door.prepareSocketIoServer (/home/andrew/enigma-bbs/core/door.js:82:11)
    at Door.run (/home/andrew/enigma-bbs/core/door.js:96:7)
    at AbracadabraModule.runDoor (/home/andrew/enigma-bbs/mods/abracadabra.js:184:16)
    at AbracadabraModule.finishedLoading (/home/andrew/enigma-bbs/mods/abracadabra.js:195:8)
    at complete (/home/andrew/enigma-bbs/mods/abracadabra.js:146:11)
    at /home/andrew/enigma-bbs/node_modules/async/dist/async.js:3861:9
    at /home/andrew/enigma-bbs/node_modules/async/dist/async.js:421:16
    at replenish (/home/andrew/enigma-bbs/node_modules/async/dist/async.js:941:25)
    at iterateeCallback (/home/andrew/enigma-bbs/node_modules/async/dist/async.js:931:17)
```

**Steps to reproduce**
Try and run a stdio door via websockets

**Comment by NuSkooler** [comment]
Damn, I'll get that fixed ASAP and pushed to master

**Comment by NuSkooler** [comment]
Fixed @ d5954f8
